_snippets/self-hosting/installation/latest-next-version.md

```diff
@@ -1,6 +1,6 @@
 /// note | Latest and Next versions
 n8n releases a new minor version most weeks. The `latest` version is for production use. `next` is the most recent release. You should treat `next` as a beta: it may be unstable. To report issues, use the [forum](https://community.n8n.io/c/questions/12){:target=_blank .external-link}.
```

Comment:
The external link syntax is incorrect. According to the style guide, external links should use quotes around `_blank`. Please change `{:target=_blank .external-link}` to `{:target="_blank" .external-link}`